src/components/TodoList/TodoItem.tsx

```diff
@@ -0,0 +1,62 @@
+import React from 'react';
+import { Todo } from '../../types/Todo';
+import classNames from 'classnames';
+import { useDispatch, useSelector } from 'react-redux';
+import { currentTodoReducer } from '../../features/currentTodo';
+import { RootState } from '../../app/store';
+
+interface Prop {
+  todo: Todo;
+}
+
+export const TodoItem: React.FC<Prop> = ({ todo }) => {
+  const dispatch = useDispatch();
+  const activeModal = () => dispatch(currentTodoReducer(todo));
+  const currentTodo = useSelector<RootState>(
+    state => state.currentTodo,
+  ) as Todo;
```

Comment:
```suggestion
  );
```
looks redundant

Reply:
for some reason TS is considering `currentTodo as unknown` , so to prevent failing I add such type assertion